csm_web/frontend/src/components/course/CourseBios.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { DayGroup } from "../../utils/queries/courses";
+import { RawUserInfo, Section, Spacetime } from "../../utils/types";
+import BioEntry from "./BioEntry";
+
+interface CourseBiosProps {
+  sectionsByDay: DayGroup[];
+}
+
+interface UserWithSectionTimes {
+  user: RawUserInfo;
+  sectionTimes: Spacetime[][];
+}
+
+const CourseBios = ({ sectionsByDay }: CourseBiosProps) => {
+  const flattenedSections = sectionsByDay.reduce((acc, curr) => acc.concat(curr.sections), [] as Section[]);
+  const users: Record<number, UserWithSectionTimes> = {};
+  const userOrder: number[] = [];
```

Comment:
Since this uses the users provided from the course sections query, a user updating their profile will not invalidate the underlying query, and the bios page will not be updated until the page is refreshed.

Ex. if you update someone's bio, then navigate to the bios page without refreshing, the update will not show.